docs/learn/advanced/security.md

```diff
@@ -20,7 +20,9 @@ The *linked commit DAG* provides a logical ordering of events which is useful wh
 
 Different StreamTypes in Ceramic may have different conflict resolution strategies with different security properties.
 
-The *earliest anchor rule* solves any conflict in the history of a stream by picking the stream that was anchored at the earliest point in time. This enables keys to be securely revoked since someone that gains possession of an old key after it was revoked will be unable to produce a *proof-of-publication* that is earlier than the first anchor. A drawback with this strategy is that the controller of the stream can create anchored commits in secret and reveal them at a later point in time to change the history of the stream. However, this can be mitigated by allowing the log to have multiple branches of history and merge them using a CRDT. This strategy is used by the [TileDocument](../../streamtypes/tile-document/overview.md).
+The *earliest anchor rule* solves any conflict in the history of a stream by picking the stream that was anchored at the earliest point in time. This enables keys to be securely revoked since someone that gains possession of an old key after it was revoked will be unable to produce a *proof-of-publication* that is earlier than the first anchor. This is used in the [3ID DID Method](../../authentication/3id-did/method.md) which registers a set of public keys associated to a DID URL, when the 3ID stream is updated with new public keys, revoking the old keys, an outside observer can be sure of the state of the 3ID without any secondary sources of information. 3IDs are built on top of the [TileDocument](../../streamtypes/tile-document/overview.md) stream type, and alternative DID methods could also be built on top of it.
+
+A drawback with this strategy is that the controller of the stream can create anchored commits in secret and reveal them at a later point in time to change the history of the stream. There are various ways to mitigate this. One of them is to only allow anchor services that actively make sure that the update is valid before it's being anchored.
```

Comment:
```suggestion
A drawback with the "earliest anchor rule" is that the controller of the stream can create anchored commits in secret and reveal them at a later point in time to change the history of the stream. There are various ways to mitigate this. One of them is to only allow anchor services that actively publish all created anchor commits when they are created.
```

Reply:
the issue is keeping the commits secret, not that they were invalid at the time that they were created, right?